test/e2e/wmco_test.go

```diff
@@ -15,6 +15,9 @@ var (
 	nodeRetryInterval    = time.Minute * 1
 	cleanupRetryInterval = time.Second * 1
 	cleanupTimeout       = time.Second * 5
+	// deploymentRetries is the amount of time to retry creating a Windows Server deployment, to compensate for the
+	// time it takes to download the Server2019 image to the node
```

Comment:
just a thought, can we download the windows container image while initializing the virtual machine?